docs/workloads/azure/frontend/SSR/getting_started_ssr.md

```diff
@@ -162,7 +178,95 @@ npm run start
 
 This will start the app in production mode, so you can see how it would behave deployed!
 
-## Customising
+## Implementation details
+
+The template is a Server Side Rendered (SSR) implementation using:
+
+- [React.js](https://reactjs.org/) for the user interface (UI)
+- React SSR framework [Next.js](https://nextjs.org/)
+- Nextjs using a
+  [Custom Server](https://nextjs.org/docs/advanced-features/custom-server) with
+  Node.js [Express.js](https://expressjs.com/)
+  
+## To build and run using Docker
+  
+  In order to be able to build and run the webapp template, across environments
+  and as part of CI, we need to use [Docker](https://docs.docker.com/install/).
+  
+  ```bash
+  # build from the webapp directory
```

Comment:
Is better to add the comment as an instruction to the user or as a title to the code block. If included in the code block it will be copied to clipboard when copied by the user.